src/vignette/util/external_hotlinking.clj

```diff
@@ -1,24 +1,41 @@
 (ns vignette.util.external-hotlinking
-  (:require [vignette.protocols :refer :all]
+  (:require [vignette.media-types :refer :all]
+            [vignette.protocols :refer :all]
             [vignette.storage.protocols :refer :all]
             [vignette.util.thumbnail :as u]))
 
 (def force-header-name "x-vignette-force-thumb")
 (def force-header-val "1")
+(def force-webp-val "webp")
```

Comment:
If we decided to define this value, we can move it somewhere else and use it other places where we currently have strings (like https://github.com/Wikia/vignette/blob/52a6aa9ed201b7456a1971275baca6e29d4eb030/src/vignette/http/route_helpers.clj#L48).